.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   release-package:
     environment: publish
```

Comment:
I think you can remove `environment` here because only the other job actually needs the token